src/vm/tests/mod.rs

```diff
@@ -103,6 +109,16 @@ pub fn execute(s: &str) -> Value {
     vm_execute(s).unwrap().unwrap()
 }
 
+pub fn execute_against_version_and_network(
```

Comment:
If you insist on keeping this (I agree with @pavitthrap -- we just `unwrap()` in-place in every other context), then can you at least call this something else to indicate that it will panic on failure?  Maybe `execute_against_version_and_network_nofail` or something?

Reply:
I removed this function.